File: scalechain-net/src/test/scala/io/scalechain/blockchain/net/handler/PongMessageHandlerSpec.scala
package io.scalechain.blockchain.net.handler

import java.io.File

import io.netty.channel.embedded.EmbeddedChannel
import org.scalatest._

class PongMessageHandlerSpec extends MessageHandlerTestTrait with ShouldMatchers {
  this: Suite =>

  val testPath = new File("./target/unittests-PongMessageHandlerSpec/")

  "handle" should "" in {
  }
}

File: scalechain-net/src/test/kotlin/io/scalechain/blockchain/net/handler/PongMessageHandlerSpec.kt
package io.scalechain.blockchain.net.handler

import io.kotlintest.matchers.Matchers
import java.io.File

class PongMessageHandlerSpec : MessageHandlerTestTrait(), Matchers {

  override val testPath = File("./target/unittests-PongMessageHandlerSpec/")

  init {
    "handle" should "" {
    }
  }
}